embedding/cpp/elm_flutter_view.cc

```diff
@@ -54,13 +56,32 @@ bool ElmFlutterView::RunEngine() {
 }
 
 void ElmFlutterView::Resize(int32_t width, int32_t height) {
-  if (view_ && (width_ != width || height_ != height)) {
-    width_ = width;
-    height_ = height;
-    FlutterDesktopViewResize(view_, width_, height_);
+  assert(IsRunning());
+
+  int32_t view_width, view_height;
```

Comment:
Could you please initialize local variable?

Reply:
i added initialize